src/core/crash-handler.c

```diff
@@ -113,7 +113,10 @@ _noreturn_ static void crash(int sig, siginfo_t *siginfo, void *context) {
                                         log_struct(LOG_EMERG,
                                                    LOG_MESSAGE("Caught <%s> from PID "PID_FMT".", signal_to_string(sig), siginfo->si_pid),
                                                    "MESSAGE_ID=" SD_MESSAGE_CRASH_PROCESS_SIGNAL_STR);
-                        }
+                        } else
+                                log_struct(LOG_EMERG,
+                                                   LOG_MESSAGE("Caught <%s>, from signal code %d.", signal_to_string(sig), siginfo->si_code),
+                                                   "MESSAGE_ID=" SD_MESSAGE_CRASH_PROCESS_SIGNAL_STR);
```

Comment:
Lines two and three should be indented to the column with `(`, like above.

Printing the numerical code is not great. A table with names for codes would be nice… I don't know how hard it would be to construct this though.